editors/code/src/extension.ts

```diff
@@ -89,13 +89,9 @@ export function activate(context: vscode.ExtensionContext) {
     }
 
     // Notifications are events triggered by the language server
-    const allNotifications: Iterable<
```

Comment:
The original indentation here is arguably bad, and we've kept running into problems with this one and the other. My version of `prettier` picks a different indentation than the one on CI.